web-components/src/components/modal/BasketPutModal.tsx

```diff
@@ -6,7 +6,7 @@ import { FormModalTemplate } from './FormModalTemplate';
 
 interface BasketPutModalProps extends RegenModalProps, BasketPutProps {}
 
-export const title = 'Put in basket';
+export const BASKET_PUT_TITLE = 'Put in basket';
```

Comment:
Change all title variables in modal with:
- specifics naming to prevent renaming in parents component
- constant casing